verify-sna/.owners

```diff
@@ -0,0 +1,2 @@
+anishshandilya
+# Insert your Github username here

```

Comment:
Why do we have this? Can this be updated to the correct owner as Anish has left Twilio?

Reply:
Done. Thanks for pointing it out.